packages/amesos2/test/adapters/Tpetra_CrsMatrix_Adapter_UnitTests.cpp

```diff
@@ -435,9 +432,6 @@ namespace {
     RCP<const Comm<int> > comm = Tpetra::getDefaultComm();
     const size_t numprocs = comm->getSize();
     const size_t rank     = comm->getRank();
-    // create a Map for our matrix
-    global_size_t nrows = 6;
```

Comment:
@iyamazaki does commenting this portion of the test remove code coverage? Is this necessary to get the changes working with Cuda-enabled SLUDist?

Reply:
I don't think this will change the test coverage, @ndellingwood.